### ISSUE TITLE
Design Intent - Accessibility Feedback - OCTO Health Apps, MHV Medical Records, After Visit Summary

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#57660]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- In the meeting Martha said I would say this if I had been there, but I'll still say it now: Thank you for prioritizing accessibility and thinking about it early!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Must:** To the extent that you have control over the after visit content, make sure the heading hierarchy makes sense.
- **Consider:** Echoing Matt's thoughts on this being a giant massive page of text... I think this would benefit from some kind of design work that makes the content easier to explore. I'm not sure that I love Matt's suggestion of using accordions, but I agree that the existing On This Page component might not be up to the task. Maybe some kind of On This Page type table of contents that's more than just H2s? Some kind of description helper text with each On This Page link to summarize what's in each of those sections? 
   
   My thinking here is that a user is probably not going to have a clear idea of what's in each of those top-level H2 sections without having to do a lot of self-guided exploration. For example, "New orders from this visit" sits under "My treatment plan" which makes sense, but I could imagine expecting to see it under "Today's visit" as well. Anything that can help guide users to the right part of the page faster (or otherwise help build some intuition about the structure of the content) would be valuable.
   
   I really don't have a specific design recommendation and so long as the heading structure makes sense I don't think there will be any technical WCAG violations. But this might be a good experimental design opportunity for exploring alternatives to or extensions of the On This Page component a bit.

**Comment by kristenmcconnell** [comment]
We tried out accordions and think that they may work to help break up the information. The focus of our usability study will be on the IA and headers of the page. Therefore, the study should give us insight into most of your concerns. We plan to use feedback to make necessary changes to the underlying AVS system.
